cli/main.go

```diff
@@ -173,9 +172,33 @@ func getEnvInt(key string, defaultValue int) int {
 	return defaultValue
 }
 
-func parseRelayURLs(relayURLs string) []server.RelayEntry {
-	ret := []server.RelayEntry{}
-	for _, entry := range strings.Split(relayURLs, ",") {
+// Deletes whitespace from beginning and end, removes empty strings
+func trim(stringSlice []string) []string {
+	filtered := make([]string, 0, len(stringSlice))
+	for _, entry := range stringSlice {
+		trimmed := strings.TrimSpace(entry)
+		if trimmed != "" {
+			filtered = append(filtered, trimmed)
+		}
+	}
+	return filtered
+}
+
+// Returns a slice of unique items
+func unique(stringSlice []string) []string {
+	filtered := make([]string, 0, len(stringSlice))
```

Comment:
```suggestion
	filtered := []string{}
```

the final size is not known, don't initialize it

Reply:
See my other comment.